web/styles/left_sidebar.css

```diff
@@ -1876,11 +1854,18 @@ li.topic-list-item {
     }
 }
 
-.direct-messages-list-filter,
-.stream-list-filter {
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow: hidden;
+.direct-messages-search-section {
+    .direct-messages-list-filter {
+        grid-area: filter-box;
+        padding-right: var(--line-height-sidebar-row-prominent);
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
```

Comment:
I wonder if these overflow styles are things that should be on the common class?